### ISSUE TITLE
Setting to play a tone with a configurable amount of seconds before the Sleep Timer ends

### ISSUE THREAD

**Issue by HStep20** [advplyr/audiobookshelf-app#600]
### Is your feature request related to a problem? Please describe.
When listening to an audiobook with a sleep timer, the behavior now just fades out until it turns off. This leaves it a bit difficult to recognize when the sleep timer is coming to an end for the phone to be shaken.

### Describe the solution you'd like
A Configurable setting to play a tone _X_ seconds before the sleep timer ends would alleviate this by giving clear audible feedback to the user which will let them know their timer is about to end and they should shake it. 

I currently use the android app [Sleep Timer](https://play.google.com/store/apps/details?id=ch.pboos.android.SleepTimer&gl=US) for an external sleep timer app which has a feature like this, if you prefer an example. It does not have a configurable setting for when to play the tone, but with as configurable as ABS seems to be going for, I think letting the user set If the tone should play, and When the tone should play in relation to the end of the sleep timer would be great.

**Comment by advplyr** [comment]
I'm testing out this app you linked and not seeing any setting to play a tone

**Comment by HStep20** [comment]
I believe it is on by default? 
It plays a tone 60 seconds before the timer ends, and plays a tone when you shake it to extend, so that you know its extended.

**Comment by DaVoodoo** [comment]
Yes, Smart AudioBook Player is where I came across this feature. 

I would love it if AudioBookShelf would implement this.

**Comment by advplyr** [comment]
A sleep timer setting was added in v0.9.81 allows for playing a chime sound when 30 seconds are remaining on the timer (android only).
It is off by default but if a lot of users end up preferring it then it can be updated to be on by default.
